Configuration/NodeTypes.yaml

```diff
@@ -34,6 +35,7 @@
     nodeCreationHandlers:
       promotedElements:
         factoryClassName: 'Neos\Neos\Ui\Infrastructure\ContentRepository\CreationDialog\PromotedElementsCreationHandlerFactory'
+    moveNodeStrategy: scatter
```

Comment:
as discussed today for content, especially as scatter also moves virtual variants the behaviour is correct

> Still, specializations pointing to the same node using the fallback mechanism will be kept gathered.